### PR TITLE
[DOCS] Building block detection alerts hidden in Security Overview page

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1182.

Previews:
* [View building block alerts in the UI](https://security-docs_1226.docs-preview.app.elstc.co/guide/en/security/master/building-block-rule.html#_view_building_block_alerts_in_the_ui) - New intro
* [View and filter detection alerts](https://security-docs_1226.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#detection-view-and-filter-alerts) - Revised last bullet point

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
LGTM!

**Review by banderror — APPROVED** [review]
LGTM, thank you 👍

**Review by benironside — APPROVED** [review]
Looks good to me. 

Not sure but maybe it would be helpful to link to the Building Block rules page that you changed in this PR from where it says "building block alerts" in the updated part of the Manage detection alerts page, to help people quickly figure out how exactly to include them in the Alerts page.

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
...
 * Visualize alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
 * Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the Count table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
-* Filter alert results to include building block alerts or to only show alerts from indicator match rules, by selecting the *Additional filters* drop-down. By default, building block alerts are excluded from the Alerts table; including them expands the number of alerts.
+* Filter alert results to include building block alerts or to only show alerts from indicator match rules, by selecting the *Additional filters* drop-down. By default, building block alerts are excluded from the Overview and Alerts pages. You can choose to include building block alerts on the Alerts page, which expands the number of alerts.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:28]:
```suggestion
* Filter alert results to include building block alerts or to only show alerts from indicator match rules, by selecting the *Additional filters* drop-down. By default, <<building-block-rules, building block alerts>> are excluded from the Overview and Alerts pages. You can choose to include building block alerts on the Alerts page, which expands the number of alerts.
```

**Comment by joepeeples** [comment]
> Not sure but maybe it would be helpful to link to the Building Block rules page that you changed in this PR from where it says "building block alerts" in the updated part of the Manage detection alerts page, to help people quickly figure out how exactly to include them in the Alerts page.

@benironside Sounds good to me. I added an internal link to the "About building block rules" page.

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Pull request merged**